flow-typed/Claim.js

```diff
@@ -191,32 +191,3 @@ declare type ChannelPublishParams = {
   tags?: Array<string>,
   languages?: Array<string>,
 };
```

Comment:
- Copy someone else's playlist -- name it "hahaha"
- Try to publish:
    - :white_check_mark:  `master`:  `name` and `title` correctly populated it as `hahaha`
    - :small_red_triangle:  `pr`:  `name` and `title` blank.  The Submit button is not grayed out (but I think it shouldn't be blank)